### PR TITLE
Feature/externalize stratus

### PR DESCRIPTION
**Pull request by apocist**
Allow use of Stratus outside of Sitetheory relative pathing

There are many CSS and custom sitetheory references, but it's progress

### CODE REVIEW

File: source/prototype.js [hunk 1]
```diff
@@ -46,6 +46,9 @@ var Stratus = {
   BaseUrl: (requirejs && _.has(requirejs.s.contexts._, 'config')
     ? requirejs.s.contexts._.config.baseUrl
     : null) || '/',
+  BundlePath: (requirejs && _.has(requirejs.s.contexts._, 'config')
+    ? requirejs.s.contexts._.config.bundlePath
```
Comment (alexgurrola) [inline source/prototype.js]: The variable bundlePath would exist in `require.js`, as it isn't a standard location.  You may want to find it in the `boot` loader.

Comment (apocist) [reply source/prototype.js]: The `requirejs.s.contexts._.config.bundlePath` loads the boot config from https://github.com/Sitetheory/stratus/pull/115/files#diff-1d77519ea8fd4dc2aead204d16d8a519R17 which is injected from any boot variables. 

I don't believe stratus.js or prototype.js can directly read the boot, other than through the config, right?

Comment (alexgurrola) [reply source/prototype.js]: The variable `boot` is global, so things like `boot.bundle` or `boot.relative` exist, but once `require.js` gets its hands on it, the object ends up becoming something else entirely, leaving only the `baseUrl` to work with.  Use `console.log(boot, requirejs.s.contexts._.config)` to compare.

Comment (apocist) [reply source/prototype.js]: the boot and config are different. in this case for my example `boot.bundle` = `content/common/` and `requirejs.s.contexts._.config.bundlePath` = `content/common/stratus/` as was intended.

To follow the same approach as the line above it (`baseUrl`)  used the config, as the config performs actions on the variables. Is there a reason it shouldn't work like the `baseUrl` does?

Comment (alexgurrola) [reply source/prototype.js]: I didn't realize `require.js` would let it through until I saw your working test.  The variable `baseUrl` is a standard config option at http://requirejs.org/docs/api.html#config and makes me wonder how much more we could throw in there down the road, as `boot` could technically not exist in the context that someone makes a completely custom config for `require.js`.  🤔

**Pull request merged**